### PR TITLE
 [svelte-leafletjs] support events props to LeafletMap

### PR DESCRIPTION
**Pull request by Ennoriel**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [source code](https://github.com/ngyewch/svelte-leaflet/blob/main/src/components/LeafletMap.svelte)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

Hi @k-yle I made a PR according to our discussion here: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/60263

### CODE REVIEW

File: types/svelte-leafletjs/index.d.ts [hunk 1]
```diff
@@ -76,9 +76,12 @@ export namespace LeafletContext {
     }
 }
 
+type KeyEvents = Array<keyof MapEvents>;
+
 export class LeafletMap extends SvelteComponentTyped<
     {
         options?: L.MapOptions;
+	    events?: KeyEvents;
```
Comment (DangerBotOSS) [inline types/svelte-leafletjs/index.d.ts:84]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## svelte-leafletjs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/svelte-leafletjs@latest/))
was missing the following properties:
1. version
1. Control
1. control
1. Browser
1. Evented

<details>
<summary>as well as these 62 other properties...</summary>
<p>Mixin, Util, Class, Handler, extend, bind, stamp, setOptions, DomEvent, DomUtil, PosAnimation, Draggable, LineUtil, PolyUtil, Point, point, Bounds, bounds, Transformation, transformation, Projection, LatLng, latLng, LatLngBounds, latLngBounds, CRS, geoJSON, geoJson, Layer, LayerGroup, layerGroup, FeatureGroup, featureGroup, imageOverlay, VideoOverlay, videoOverlay, SVGOverlay, svgOverlay, DivOverlay, popup, tooltip, icon, DivIcon, divIcon, marker, tileLayer, GridLayer, gridLayer, SVG, svg, Renderer, Canvas, canvas, Path, circleMarker, circle, polyline, polygon, rectangle, Map, map, noConflict</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2e5bc957940b515f4ac94cdecf3583d8b1bf87a4
</p>

**Review by k-yle — APPROVED** [review]
This is great, thanks @Ennoriel. Sorry I haven't had time to do this myself. 

`events` seems to apply to everything, not just `LeafletMap`, but we can add that in a follow up PR if you prefer.

**Review by k-yle — APPROVED** [review]
looks great, thanks again

**Comment by Ennoriel** [comment]
Ready to merge

**Pull request merged**